### ISSUE TITLE
instruct users not to assign a navigation key to an nvda command

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#17368]
### Is your feature request related to a problem? Please describe.
A visually impaired person and nvda user contacted us asking for help.
It turns out that every time she pressed tab, nvda displayed a confirmation window asking if she wanted to turn on the screen curtain.
I investigated the case and discovered that the tab key was assigned to start the screen curtain.
I considered this serious because she was unable to navigate the computer due to this assignment.
The user did this without realizing it (it's hard to imagine but it happened).
### Describe the solution you'd like
prevent a user from assigning navigation keys or keys that are frequently used and potentially prevent the user from performing important tasks, such as tab, shift + tab, arrow keys, enter, esc, space, backspace.
These are just suggestions; you can modify the keys and combinations as you wish.
If the case is not preventing, at least a warning window will be displayed informing that the user may be unable to navigate in some ways if this key is assigned as an nvda command.
### Describe alternatives you've considered
warning of a risk involved, or preventing the key from being assigned.
I would also consider this for add-ons. I recently found an add-on that assigned ctrl to open a window. This was not a big problem, more of an inconvenience. It would be worth warning the user that he or an add-on assigned a key that is so important to one of his functions, and that this could cause navigation problems, or at least an inconvenience to the user.
### Additional context
This may be irrelevant for more knowledgeable users, but not for beginners.

**Comment by seanbudd** [comment]
We would accept adding a warning when assigning certain keys in the dialog. We don't think users should be prevented from making modifications as necessary
